### PR TITLE
Fix today task list to only show today's tasks 

### PR DESCRIPTION
**Pull request by mikeFam**
---
name: #194 
about: Added condition to check if the current date is between task's start and end dates
title: Display only todays in it's column
labels: bug
assignees: ''
---

**Describe the issue**
This issue is a bug fix to display only tasks that are valid of the current date/day to today's task list on the home page and dashboard

**Describe the solution**
Added conditional statement to compare the timestamp of the tasks objects and the current date's timestamp, display task if the current date time stamp is between the start and the end timestamp

### CODE REVIEW

**Review by bbqnvd — CHANGES_REQUESTED** [review]
It was great to work with you Michael. The code fixed the bug in a quick and efficient way (compare the seconds). Good job! However, before it's merged, I think you need to exclude the changes made to package-lock.json as only the TaskList.jsx file is modified.

**Review by bbqnvd — APPROVED** [review]
Looks awesome!! Appreciate your hard work!

File: src/components/global/TaskList.jsx [hunk 1]
```diff
@@ -14,25 +15,35 @@ function TaskList(props) {
...
-                selected={currentTask && t.Task_id === currentTask.Task_id}
-                endDate={t.End_date}
-                label={t.Label}
-                startDate={t.Start_date}
-                taskId={t.Task_id}
-                checked={t.Is_complete}
-                description={t.Description}
-                userId={t.User_id}
-              />
+              {t.Start_date &&
```
Comment (Denijar) [inline src/components/global/TaskList.jsx:31]: Currently you are able to successfully create a task without an end or start date. This new logic will mean that even though that tasks gets persisted in the database, it won't ever show in the task list. This is not that great.

**Review by Denijar — APPROVED** [review]
Just one observation that I don't think needs to be addressed to merge. However, if you have time, perhaps the `handleAddNewTask` function in `NewTask.jsx` could be extended to add a check that the task being added has a start and end time to remove this quirk.

File: src/components/global/TaskList.jsx (hunk 1 above)
Comment (mikeFam) [reply src/components/global/TaskList.jsx:31]: Nice catch, was an easy fix, Thanks Denise :)

**Pull request merged**